test/spark/app/build.sbt

```diff
@@ -0,0 +1,11 @@
+name := "sonnets"
+
+version := "0.1"
+
+scalaVersion := "2.12.10"
+
+libraryDependencies ++= Seq(
+    "org.apache.spark" %% "spark-sql" % "3.1.1" % "provided",
```

Comment:
Should we consider adding another system test that runs a 2.x.x Spark application? 
@arielshaqed is this needed?

Reply:
Yes, but in the next PR :-)